K2fov/fov.py

```diff
@@ -7,6 +7,8 @@
     import matplotlib
 except ImportError:
     pass
+from astropy.coordinates import SkyCoord
+from astropy import units as u
```

Comment:
I'm going to move the AstroPy imports inside `pickAChannelList` to prevent AstroPy from being a requirement to run the existing K2fov features for now.